web/docs/project/env-vars.md

```diff
@@ -92,40 +93,44 @@ These files should not be committed to version control, and they are already ign
 ### 2. Using Shell
 If you set environment variables in the shell where you run your Wasp commands (e.g., `wasp start`), Wasp will recognize them.
 
-You can set environment variables in the `.profile` or a similar file, or by defining them at the start of a command:
-
-```shell
-SOME_VAR_NAME=SOMEVALUE wasp start
-```
+You can set environment variables in the `.profile` or a similar file, which will set them permanently, or you can set them temporarily by defining them at the start of a command (`SOME_VAR_NAME=SOMEVALUE wasp start`).
 
- This is not specific to Wasp and is simply how environment variables can be set in the shell.
+This is not specific to Wasp and is simply how environment variables can be set in the shell.
 
-Defining environment variables in this way can be cumbersome even for a single project and even more challenging to manage if you have multiple Wasp projects. Therefore, we do not recommend this as a default method for providing environment variables to Wasp projects. However, it can be useful for occasionally **overriding** specific environment variables because environment variables set this way **take precedence over those defined in `.env` files**.
+Defining environment variables in this way can be cumbersome even for a single project and even more challenging to manage if you have multiple Wasp projects. Therefore, we do not recommend this as a default method for providing environment variables to Wasp projects during development, you should use .env files instead. However, it can be useful for occasionally **overriding** specific environment variables because environment variables set this way **take precedence over those defined in `.env` files**.
 
 ## Defining Env Vars in Production
 
-While in development, we had the option of using `.env` files which made it easy to define and manage env vars. However, in production, we need to provide env vars differently.
+While in development, we had the option of using `.env` files which made it easy to define and manage env vars. However, in production, when our code is built and deployed, we need to provide env vars differently.
 
 ![Env vars usage in development and production](/img/env/prod_dev_fade_2.svg)
 
 ### Client Env Vars
 
-Client env vars are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys).
+Client env vars are embedded into the client code during the build process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys).
```

Comment:
Again, the API key is I believe an incorrect example (as it's visible by anyone once the site is deployed).

Reply:
I don't get what you mean. We are telling them they shouldn't provide secrets as clienv env vars, specifically taking an example of secret API keys. I am not sure how that refers to what you said.

Reply:
I'm resolving this because the same discussion is happening somewhere else.